includes/documents/abstract-wcpdf-order-document-methods.php

```diff
@@ -916,10 +916,10 @@ public function get_woocommerce_totals() {
 		foreach ( $totals as $key => $total ) {
 			$label = $total['label'];
 			$colon = strrpos( $label, ':' );
-			if( $colon !== false ) {
+			if ( $colon !== false ) {
 				$label = substr_replace( $label, '', $colon, 1 );
 			}
-			$totals[$key]['label'] = $label;
+			$totals[ $key ]['label'] = __( $label, 'woocommerce-pdf-invoices-packing-slips' );
```

Comment:
You need to follow this approach (textdomain included): https://github.com/wpovernight/woocommerce-pdf-ips-templates/commit/4af49ec3491f23330888a1a7e8f6c31bb1e1d1b2#diff-e54d4a86c56437f82e078cf79de1abf3c9e5a4c18bb2a76170c08f09ec563d24R356-R365

Reply:
Thanks for the suggestion. Updated here: https://github.com/wpovernight/woocommerce-pdf-invoices-packing-slips/pull/831/commits/8be4bf92c5f856989ecc4fab41520e6729f6875e